src/content/whats-new/2024/02/whats-new-02-27-compute-add-on-sku

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Launching the Compute Add On SKU'
+summary: 'Unlock access to all new capabilities, including Live Archives and IAST, with a single SKU'
+releaseDate: '2024-02-27'
+learnMoreLink: 'https://docs.newrelic.com/docs/licenses/license-information/usage-plans/new-relic-usage-plan/#list-price'
+---
+
+The Compute Add On enables you to configure and use all of New Relic’s newest innovations, including our recently-launched Live Archives and IAST, while remaining on the user model.
```

Comment:
Updated, thanks @rhetoric101 ! For the second piece, I'd prefer to just say "data and user" model rather than given additional details, if that suits you. I've updated that in the Google doc.